server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -752,10 +752,11 @@ cics_transform_xmltodata: XMLTODATA (CHANNEL cics_data_value | DATCONTAINER cics
 /** UNLOCK */
 cics_unlock: UNLOCK cics_file_name (TOKEN cics_data_area | SYSID cics_data_area | cics_handle_response)*;
 
-/** UPDATE COUNTER / DCOUNTER */
+/** UPDATE COUNTER / UPDATE DCOUNTER */
 cics_update: UPDATE (cics_update_counter | cics_update_dcounter);
-cics_update_counter: COUNTER cics_name (POOL cics_name | VALUE cics_data_value | COMPAREMIN cics_data_value | COMPAREMAX cics_data_value | cics_handle_response)+;
-cics_update_dcounter: DCOUNTER cics_name (POOL cics_name | VALUE cics_data_area | COMPAREMIN cics_data_area | COMPAREMAX cics_data_area | cics_handle_response)+;
+cics_update_counter: ((COUNTER | POOL) cics_name | (VALUE  | COMPAREMIN  | COMPAREMAX) cics_data_value | NOSUSPEND | cics_handle_response)*;
+cics_update_dcounter: ((DCOUNTER | POOL) cics_name | (VALUE  | COMPAREMIN  | COMPAREMAX) cics_data_area | NOSUSPEND | cics_handle_response)*;
```

Comment:
These rules are identical up to (D)COUNTER - merge them into one and implement the COUNTER/DCOUNTER exclusive check.
Also at least one parameter is required (`*` -> `+`)